tools/proofers/preview.js

```diff
@@ -66,7 +66,6 @@ function removeTrail(txt) {
 
 const blockType = {
     PARA: 0,
-    SECHEAD: 1,
```

Comment:
since we removed this entry, should we renumber the later ones?

Reply:
I don't think we need to since they are just arbitrary numbers.

Reply:
No, definitely not required, so makes sense to leave as it to reduce risk of change.